content/6_Advanced/LCT.mdx

```diff
@@ -355,16 +388,226 @@ int main() {
 }
 ```
 
-## Link Cut Tree - Connectivity
+## Finding Connectivity
+
+The link cut tree can be used to solve problems that deal with queries updating trees and querying for connectivity.
+
+### Problems
 
 <Problems problems="con" />
```

Comment:
@ryanchou-dev this part is just two problems...i'm thinking smth like either delete both or keep just one and explain it

Reply:
@Gabriel-Xu are you cool with explaining one of these problems briefly?

i think explanation would be better than just deleting the section

Reply:
> @Gabriel-Xu are you cool with explaining one of these problems briefly?
> 
> i think explanation would be better than just deleting the section

Do you want it to be another focus problem?

Reply:
Also does this mean I should include a code solution or just a text explanation?

Reply:
For now, I added a brief explanation.